components/prompt/report-modal.tsx

```
@@ -0,0 +1,71 @@
+}: ReportModalProps) {
+  const reportIsDisabled =
+    !reason || (reason === "OTHER" && !reasonDetail.trim());
+  return (
+    <div className="flex flex-col gap-4">
+      <div className="flex justify-center items-center heading-small">
+        {title}
+      </div>
+      <div className="flex flex-col">
+        <RadioGroup value={reason} onValueChange={onSelect}>
+          {REASON_TYPE.map((r) => (
+            <BaseRadioItem label={r.label} id={r.value} value={r.value} />
```

Biome (2.3.14)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)